[LTS] WIA and KA alignment (#1139)
italia/eidas-it-wallet-docs: docs/en/wallet-attestation-issuance.rst

..   Sequence Diagram for Wallet App Attestation acquisition

**Step 1**: The User initiates a new operation that necessitates the acquisition of a Key Attestation.

**Steps 2-3**: The Wallet Instance MUST:

  1. Verify the existence of Cryptographic Hardware Keys. If none exist, Wallet Instance re-initialization is required (:ref:`WP_140a <wallet-instance-optional-testcases>`).

>   2. Generate one or batch of asymmetric credential key pair(s) to be attested in Key Attestation (``key_pub_1``, ``key_priv_1``, ..., ``key_pub_n``, ``key_priv_n``).

Reviewer: ```suggestion
  2. Generate one or batch of asymmetric Credential key pair(s) to be attested in Key Attestation (``key_pub_1``, ``key_priv_1``, ..., ``key_pub_n``, ``key_priv_n``).
```

*  requests the Device Integrity Service to create an ``integrity_assertion`` value for each ``client_data_hash`` per each ``key_pub``.
*  receives a signed ``integrity_assertion`` value from the Device Integrity Service, authenticated by the OEM.
*  generate ``keys_to_attest`` value by signing the ``integrity_assertion`` that is obtained for the Key Attestation using the private key of the initially generated credential key pair (``priv_key``). 

**Steps 20-21 (Key Attestation Issuance Request)**: The Wallet Instance:

> * Constructs the Key Attestation Request in the form of a JWT. This JWT includes the ``integrity_assertion``, ``keys_to_attest``, ``hardware_signature``, ``nonce``, ``hardware_key_tag``, ``cnf``, ``platform`` and other configuration related parameters (see :ref:`Table of the Key Attestation Request Body <table_ka_request_claim>`) and is signed using the private key that it is public key illustrated in the request through ``cnf`` (first element of ``keys_to_attest``) (:ref:`WP_140–141 <wallet-instance-optional-testcases>`).

Reviewer: ```suggestion
* Constructs the Key Attestation Request in the form of a JWT. This JWT includes the ``integrity_assertion``, ``keys_to_attest``, ``hardware_signature``, ``nonce``, ``hardware_key_tag``, ``cnf``, ``platform`` and other configuration related parameters (see :ref:`Table of the Key Attestation Request Body <table_ka_request_claim>`). The Key Attestation Request MUST be signed using the private key related to the public key included in the request, using the ``cnf`` parameter (first element of ``keys_to_attest``) (:ref:`WP_140–141 <wallet-instance-optional-testcases>`).
```

  8. The device in use MUST be free of known security flaws and meet the minimum security requirements defined by the Wallet Provider.
  9. The URL in the ``iss`` parameter MUST match the Wallet Provider's URL identifier (:ref:`WP_143g <wallet-instance-optional-testcases>`).

Upon successful completion of all checks, the Wallet Provider issues a Key Attestation valid for at least one month.

> **Step 28 (Key Attestation Issuance Response)**: Upon successful completion, the Wallet Provider MUST return a confirmation response using status code 200 and Content-Type ``application/json``, containing the Key Attestations signed by the Wallet Provider in the JWT format. The Wallet Instance will then perform security and integrity verification of the Key Attestations received in addition to trust verification of its Issuer (:ref:`WP_030–031 <wallet-instance-testcases>`).

Reviewer: ```suggestion
**Step 28 (Key Attestation Issuance Response)**: Upon successful completion, the Wallet Provider MUST return a confirmation response using status code set with ``200`` and Content-Type ``application/json``. The response MUST contain the Key Attestations signed by the Wallet Provider using the JWT format. The Wallet Instance MUST perform security and integrity verification of the Key Attestations received, along with the trust verification already evaluated about the Issuer (:ref:`WP_030–031 <wallet-instance-testcases>`).
```
